tests/test_landings.py

```diff
@@ -47,7 +45,7 @@ def test_landing_revision_saves_data_in_db(db, client, phabfactory):
 
     # test saved data
     landing = Landing.query.get(1)
-    assert landing.serialize() == CANNED_LANDING_1
+    assert landing.serialize() == CANNED_LANDING_FACTORY_1
```

Comment:
This is for a follow-up commit, but I don't understand what this test is verifying.  It saves two POSTs to the database, then checks the database contents.  If it's testing some sort of sequence generation across POSTs, then the test name and content should focus only on verifying that.  If it's testing POST data serialization to the database, then only that bit should be verified (and the second POST should be dropped).